RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -57,18 +57,21 @@
 RecoLocalCaloRECO.outputCommands.extend(ecalLocalRecoRECO.outputCommands)
 RecoLocalCaloAOD.outputCommands.extend(ecalLocalRecoAOD.outputCommands)
 
-def _modifyRecoLocalCaloEventContentForHGCalRECO( obj ):
-    obj.outputCommands.append('keep *_HGCalRecHit_*_*')
+# mods for HGCAL
 
-def _modifyRecoLocalCaloEventContentForHGCalFEVT( obj ):
-    obj.outputCommands.append('keep *_HGCalRecHit_*_*')
-    obj.outputCommands.append('keep *_HGCalUncalibRecHit_*_*')
+_phase2_RecoLocalCaloFEVT_outputCommands = RecoLocalCaloFEVT.outputCommands
+_phase2_RecoLocalCaloFEVT_outputCommands.append('keep *_HGCalRecHit_*_*')
+_phase2_RecoLocalCaloFEVT_outputCommands.append('keep *_HGCalUncalibRecHit_*_*')
 
+_phase2_RecoLocalCaloRECO_outputCommands = RecoLocalCaloRECO.outputCommands
+_phase2_RecoLocalCaloRECO_outputCommands.append('keep *_HGCalRecHit_*_*')
+
+_phase2_RecoLocalCaloAOD_outputCommands = RecoLocalCaloAOD.outputCommands
+_phase2_RecoLocalCaloAOD_outputCommands.append('keep *_HGCalRecHit_*_*')
 
-# mods for HGCAL
 from Configuration.StandardSequences.Eras import eras
-eras.phase2_hgcal.toModify( RecoLocalCaloFEVT, func=_modifyRecoLocalCaloEventContentForHGCalFEVT)
-eras.phase2_hgcal.toModify( RecoLocalCaloRECO, func=_modifyRecoLocalCaloEventContentForHGCalRECO )
+eras.phase2_hgcal.toModify( RecoLocalCaloFEVT, outputCommands = _phase2_RecoLocalCaloFEVT_outputCommands )
```

Comment:
I think a modified version of @makortel 's suggestion might be easy to maintain and still use `toModify`
```python
eras.phase2_hgcal.toModify( RecoLocalCaloFEVT, outputCommands = RecoLocalCaloFEVT.outputCommands + [
        'keep *_HGCalRecHit_*_*',
        'keep *_HGCalUncalibRecHit_*_*'
    ]
)